here-naksha-lib-base/src/commonMain/kotlin/naksha/base/AnyObject.kt

```diff
@@ -10,4 +10,10 @@ import kotlin.js.JsExport
  */
 @Suppress("unused", "OPT_IN_USAGE")
 @JsExport
-open class AnyObject : MapProxy<String, Any>(String::class, Any::class)
+open class AnyObject : MapProxy<String, Any>(String::class, Any::class) {
+    fun contentDeepEquals(other: AnyObject): Boolean {
+        return Platform.deepEquals(this, other)
+    }
```

Comment:
Why is this method not added to `Proxy` as `deepEquals`? It should be applicable for all types or?